utils/appliance.py

```diff
@@ -275,6 +275,43 @@ def destroy(self):
             self.remove_rhev_direct_lun_disk()
         self.provider.delete_vm(self.vm_name)
 
+    def stop(self):
+        """Stops the VM this appliance is running as
+        """
+        self.provider.stop_vm(self.vm_name)
+        self.provider.wait_vm_stopped(self.vm_name)
+
+    def start(self):
+        """Starts the VM this appliance is running as
+        """
+        self.provider.start_vm(self.vm_name)
+        self.provider.wait_vm_running(self.vm_name)
+
+    def templatize(self):
```

Comment:
I would like to see this split up across the two classes, so the "cleaning" is done in IPAppliance, but the templatize is done in Appliance.